tempora/schedule.py

```diff
@@ -158,14 +168,14 @@ class Scheduler:
     and dispatching them on schedule.
     """
 
-    def __init__(self):
-        self.queue = []
+    def __init__(self) -> None:
+        self.queue: list[DelayedCommand] = []
 
-    def add(self, command):
+    def add(self, command) -> None:
         assert isinstance(command, DelayedCommand)
```

Comment:
I'm thinking with a type annotation that the assertion can be removed.

```suggestion
    def add(self, command: DelayedCommand) -> None:
```